addons/sms/tests/common.py

```diff
@@ -183,7 +183,7 @@ def assertNoSMSNotification(self, messages=None):
         self.assertEqual(self.env['mail.notification'].search(base_domain), self.env['mail.notification'])
         self.assertEqual(self._sms, [])
 
-    def assertSMSNotification(self, recipients_info, content, messages=None, check_sms=True, sent_unlink=False):
+    def assertSMSNotification(self, recipients_info, content, messages=None, check_sms=True, sent_unlink=False, notif_content=None):
```

Comment:
nit, I first read "notif_content" as "notify_content" and I was expecting a bool, True we notify the content, False we don't, suggestion: `notification_content`. I never read/heard "notif" used as short for "notification" outside of french culture.